cmd/cloud/setup.go

```diff
@@ -269,17 +278,18 @@ func checkAPIKeys(astroClient astro.Client, args []string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-
-	organizations, err := astroClient.GetOrganizations()
+	orgs, err := organization.ListOrganizations(coreClient)
```

Comment:
The current code uses GQL services to fetch org list (which differ from organization package.. don't know why). i've switched to consume core services as well.

Reply:
Nice catch...could be legacy code that was just forgotten. Thanks for migrating it over.